python/ray/tests/test_memory_pressure.py

```diff
@@ -355,8 +356,16 @@ async def test_task_oom_logs_error(ray_with_memory_monitor):
         verified = True
     assert verified
 
-    # TODO(clarng): verify task info once state_api_client.get_task_info
-    # returns the crashed task.
+    def verify_oom_task_error():
+        tasks = list_tasks(filters=[("name", "=", "allocate_memory")])
```

Comment:
Should it return False instead for things it is waiting on ?

Reply:
If the asserts fail (it will raise exceptions), thus the waiting will fail for that iteration. So we don't have to explicitly return false.